### ISSUE TITLE
Failed to get revokefs-fuse socket from system-helper: User flatpak does not exist in password file entry

### ISSUE THREAD

**Issue by cho2** [flatpak/flatpak#2959]
## Linux distribution and version

openSUSE Tumbleweed

## Flatpak version

Flatpak 1.4.0

## Description of the problem

Run `flatpak update` and `flatpak install bla.bla.bla`

Other details https://bugzilla.opensuse.org/show_bug.cgi?id=1137537

## Steps to reproduce

For example: 

```
flatpak install flathub com.dropbox.client
Mencari kecocokan...

Remote 'flathub' ditemukan pada beberapa pemasangan:

   1) system
   2) user

Mana yang ingin Anda gunakan (0 untuk membatalkan)? [0-2]: 1
Ref ‘app/com.dropbox.Client/x86_64/stable’ ditemukan dalam remote ‘flathub’ (system):
Gunakan ref ini? [Y/n]: y

com.dropbox.Client izin:
    ipc     network     pulseaudio     x11     file access [1]

    [1] /tmp, home


        ID                           Arsitektur          Cabang          Remote           Unduh
 1. [✓] com.dropbox.Client           x86_64              stable          flathub          92,8 MB / 92,9 MB

Peringatan: Failed to get revokefs-fuse socket from system-helper: User flatpak does not exist in password file entry
Pemasangan selesai.

```
```
flatpak update
Mencari pemutakhiran...

        ID                                    Arsitektur           Cabang           Remote            Unduh
 1. [\] io.designer.GravitDesigner            x86_64               stable           flathub           40,3 MB / 384,2 MB

Peringatan: Failed to get revokefs-fuse socket from system-helper: User flatpak does not exist in password file entry
Memutakhirkan… ███▍                  17%  1,3 MB/s  02:36
```

**Comment by mattisme** [comment]
I had this error on voidlinux, I fixed it by installing the fuse3 package.

**Comment by matthiasclasen** [comment]
This sounds like a packaging problem - flatpak 1.4 packages need to create the flatpak user.

**Comment by filmor** [comment]
Is this documented somewhere? The gentoo package has the same problem. Does this user need any particular rights, own anything in particular, be part of a group or does it just need to exist?

**Comment by glaubersm** [comment]
Same thing with flatpak 1.4.1 on Void Linux.

**Comment by refi64** [comment]
If this comes up, please report it as a bug in your distro's packages.

**Comment by filmor** [comment]
Fine, it would still be interesting how to actually fix it. Is a bare nologin user without home and without any groups enough?

**Comment by refi64** [comment]
Based on the source, the only requirement seems to be that it's some user with some group ID associated with it. If in doubt, you can probably check out [Fedora's spec file](https://src.fedoraproject.org/rpms/flatpak/blob/master/f/flatpak.spec#_158) as well.

**Comment by cho2** [comment]
Fixed on Flatpak 1.4.1

**Comment by sashaaaaaaaaa** [comment]
I still get this on flatpak 1.6.2 on slackware-current.
